ex0.py

```diff
@@ -0,0 +1,19 @@
+from PIL import Image
+
+
+def decipher(image_path):
+    #  decipher the hidden message inside the image
+    black_pixels = []
+    coded_message = ""
+    threshold = 127  # lower than this is black
+    image = Image.open(image_path)
```

Comment:
Opening the image and the logic of the function doesnt have anything to do with eachother
Seprate IO from logic  - 
https://wiki.c2.com/?SeparateIoFromCalculation